### PR TITLE
Add extra CSS styling and layout, and Image Sketch

### PR DESCRIPTION
**Pull request by lornaarmstrong**
Add extra CSS to make the fixed footer, 'job-role' banner. Also, add the image of Edinburgh and the description.

Note: The layout of the buttons and the 'click here to get a random message' box has it's problems, but this was not the priority of this commit, and will be acted upon in due course.

Here's a screenshot showing the changes:
Screenshot - Sketch Added

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,41 +8,60 @@
...
-          <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
-          <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
-          <button class="hyperlinked-button buttonGray">GitHub</button></a>
+        <a href="https://www.linkedin.com/in/lornafarmstrong" target="_blank">
+        <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
+        <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
+        <button class="hyperlinked-button buttonGray">GitHub</button></a>
       </div>
+      </section>
       <footer>lornaarmstrong</footer>
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/index.html:65]: Would be nice to leave a TODO comment on top of this footer to highlight that it's not currently shown as expected and you will fix it later. [Example](https://www.jetbrains.com/help/idea/using-todo.html)

Comment (MostafaGabriel) [inline portfolio/src/main/webapp/index.html:65]: Would be nice to leave a TODO comment on top of this footer to highlight that it's not currently displayed as expected and you will fix it later. [Example](https://www.jetbrains.com/help/idea/using-todo.html)

Comment (MostafaGabriel) [inline portfolio/src/main/webapp/index.html:65]: Would be nice to leave a TODO comment on top of this footer to highlight that it's not currently shown as expected and you will fix it later. [Example](https://www.jetbrains.com/help/idea/using-todo.html)

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -40,13 +67,28 @@ footer {
...
+}
 
-img {
-  padding: 5px;
-  margin: 15px;
-  border: 2px outset;
+.img-nonprofile {
+  padding: 0px;
+  margin: 0px;
+  border: 0px;
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/style.css:91]: I guess 0 should be the default value for the CSS class properties, try to remove it and see if it stays the same

Comment (lornaarmstrong) [reply portfolio/src/main/webapp/style.css:91]: Thanks! I removed it and yes, it stays the same.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html:65]: Thanks!

**Pull request merged**